packages/react/src/components/ContextualMenu/ContextualMenu.types.ts

```diff
@@ -542,6 +542,11 @@ export interface IContextualMenuItem {
    */
   keytipProps?: IKeytipProps;
 
+  /**
+   * @deprecated Use subMenuProps.items instead.
+   */
+  items?: IContextualMenuItem[];
```

Comment:
We had a build break due to this change. Ironically any property optionally added to this interface will break consumers due to the `[propertyName: string]: any;` property below. @ecraig12345 fyi